mne/channels/channels.py

```diff
@@ -560,14 +560,18 @@ def inst_has(attr):
         elif isinstance(self, Evoked):
             self.data = self.data.take(idx, axis=0)
 
-    def add_channels(self, add_list, copy=None):
+    def add_channels(self, add_list, force_update_info=False, copy=None):
```

Comment:
should probably put after `copy` even though it's about to be removed (could in 0.12 break someone's code)